pipelineBackends/tokenizer.go

```diff
@@ -17,19 +17,26 @@ type Tokenizer struct {
 }
 
 func LoadTokenizer(model *Model, s *options.Options) error {
-	tokenizerBytes, err := util.ReadFileBytes(util.PathJoinSafe(model.Path, "tokenizer.json"))
-	if err != nil {
-		return err
-	}
+	if exists, err := util.FileExists(util.PathJoinSafe(model.Path, "tokenizer.json")); err == nil {
```

Comment:
Now that tokenizers are optional, we should have a standardised check in pipelines that require tokenizers, to confirm we get an appropriate error there if the file is missing